templates/mixins/flash-messages.pug

```diff
@@ -3,7 +3,5 @@ mixin flash-messages(messages)
     div.alert(class=flash['class'])
       ul.flash-errors
         if flash.messages
-          if flash.messages[0].msg
+          if flash.messages[0].msg 
             li <strong>#{flash.type}</strong> !{flash.messages[0].msg}
```

Comment:
I would still add the if flash.messages[0]
to protect when the object is undefined